models/village.settings.json

```diff
@@ -27,14 +27,12 @@
       "default": true
     },
     "state": {
-      "via": "villages",
-      "plugin": "crm-plugin",
-      "model": "state"
+      "model": "state",
```

Comment:
a 2 way relationship is added since in some projects there are filters for example: village module has filters for state and district and with a 2 way relation this is possible.

Reply:
Kurund already had explained me this on call the other day. Sorry about that, forgot to revert the change. You can discard this.